src/components/modules/Error/Error.view.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <message
+    :error="true"
+    data-test="error-message"
```

Comment:
Are you sure about that? What about cases when we should have multiple errors in the one component? I think `data-test` should not be in reusable components, because finally, we would test component where this component would be used

Reply:
didn't find any usage of `error-message` will remove this